conf/recordmanager.ini.sample

```diff
@@ -32,18 +32,28 @@ unicode_normalization_form = NFKC
 ; Wait time between request attempts in seconds. Default is 30.
 ;retry_wait = 120
 
-[Mongo]
-; url = mongodb://{username}:{password}@localhost
-;url = mongodb://localhost
-; Use a unix socket like below, if possible. It can be an order of magnitude faster than TCP/IP.
-url = "mongodb://%2Ftmp%2Fmongodb-27017.sock"
-; Database to use
+[Database]
+; URL for a MongoDB database.
+; Use a unix socket like below, if possible. It can be an order of magnitude faster
+; than TCP/IP.
+connection = "mongodb://%2Ftmp%2Fmongodb-27017.sock"
+;connection = mongodb://{username}:{password}@localhost
+;connection = mongodb://localhost
+; MongoDB database to use
 database = recman
```

Comment:
If this is a Mongo-specific setting, seems like renaming it to reduce confusion might be a good idea. Or alternatively, do you have separate [Mongo] and [PDO] sections, and then pick which one is used through a  [Database] section? That might make the overall configuration less potentially confusing, especially if more backends are introduced in future.

Reply:
Good idea. Now separated. Added bonus is that the MongoDB config is now fully backwards compatible without any tweaks.

Reply:
Great, glad I could help!